features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/mbedtls_device.h

```diff
@@ -21,7 +21,13 @@
 #define MBEDTLS_DEVICE_H
 
 #ifdef TARGET_UBLOX_EVK_ODIN_W2
+
 #define MBEDTLS_MPI_WINDOW_SIZE     3 /**< Maximum windows size used. */
+
+#else
+
+#define MBEDTLS_AES_ALT
```

Comment:
Could you please add a code comment as to why TARGET_UBLOX_EVK_ODIN_W2 doesn't use hardware AES? This will help if anyone later wants to enable and test it, or to let folks know this may be a hardware limitation of that target and thus should never be turned on.

Reply:
Workaround since #10088

Reply:
> Workaround since #10088

@jeromecoutant What does this mean, no comment needed here?

Reply:
I changed my mind. I am going to remove this update.

I am currently working on mbedtls usecases,
so when this PR will be integrated, I will push a good correction for #10088

Reply:
@Patater please approve this PR

Note I will request more review from you soon :-)